src/Fantomas.Core/SyntaxOak.fs

```diff
@@ -1348,6 +1348,7 @@ type ElseIfNode(mElse: range, mIf: range, condition: Node, range) as elseIfNode
 
         member _.AddAfter(triviaNode: TriviaNode) =
             match triviaNode.Content with
+            | TriviaContent.Newline -> condition.AddBefore triviaNode
```

Comment:
I think it makes more sense to bring this into the `ifNode`.
Something like:
```fsharp
            member _.AddBefore(triviaNode: TriviaNode) =
                match triviaNode.Content with
                | CommentOnSingleLine _
                | Newline -> condition.AddBefore triviaNode
                | _ -> (elseIfNode :> Node).AddAfter triviaNode
```

It hits the `AddBefore` of the `ifNode` first and it makes sense to apply the same logic as for `CommentOnSingleLine` there.